prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -409,4 +409,23 @@ periodics:
       - "-zone=us-west1-a"
       - "-images=projects/bct-prod-images/global/images/family/debian-10,projects/bct-prod-images/global/images/family/debian-11,projects/bct-prod-images/global/images/family/centos-7,projects/bct-prod-images/global/images/family/centos-stream-8,projects/bct-prod-images/global/images/family/centos-stream-9,projects/bct-prod-images/global/images/family/rhel-7,projects/bct-prod-images/global/images/family/rhel-8,projects/bct-prod-images/global/images/family/rhel-9,projects/bct-prod-images/global/images/family/rocky-linux-8,projects/bct-prod-images/global/images/family/rocky-linux-8-optimized-gcp,projects/bct-prod-images/global/images/family/almalinux-8"
       - "-test_projects=compute-image-test-pool-002,compute-image-test-pool-003,compute-image-test-pool-004,compute-image-test-pool-005"
-      - "-exclude=oslogin"
+      - "-exclude=(oslogin)|(networkperf)|(storageperf)"
+- name: cit-periodics-performance
+  cluster: gcp-guest
+  decorate: true
+  annotations:
+    testgrid-dashboards: googleoss-gcp-guest
+    testgrid-tab-name: cit-periodics-performance
+  interval: 12h
+  spec:
+    containers:
+    - image: gcr.io/compute-image-tools/cloud-image-tests:latest
+      imagePullPolicy: Always
+      command:
+      - "/manager"
+      args:
+      - "-project=gcp-guest"
+      - "-zone=us-central1-c"
+      - "-images=projects/bct-prod-images/global/images/family/debian-10,projects/bct-prod-images/global/images/family/debian-11,projects/bct-prod-images/global/images/family/centos-7,projects/bct-prod-images/global/images/family/centos-stream-8,projects/bct-prod-images/global/images/family/centos-stream-9,projects/bct-prod-images/global/images/family/rhel-7,projects/bct-prod-images/global/images/family/rhel-8,projects/bct-prod-images/global/images/family/rhel-9,projects/bct-prod-images/global/images/family/rocky-linux-8,projects/bct-prod-images/global/images/family/rocky-linux-8-optimized-gcp,projects/bct-prod-images/global/images/family/almalinux-8"
+      - "-test_projects=compute-image-test-pool-002,compute-image-test-pool-003,compute-image-test-pool-004,compute-image-test-pool-005"
+      - "-filter=(networkperf)|(storageperf)"
```

Comment:
Do you really want to run oslogin test with this one?

Reply:
With the filter option, it only runs the ones in filter, and ignores all other test suites.

With this setting, it would only run networkperf and storageperf, and not run oslogin.

Reply:
Ah! true, I keep misreading the `-filter` option.